RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Release
+
+## On a fork
+
+Please follow these instructions carefully.
+Ensure that you name the branch precisely as `release-vX.Y.Z`
+since this nomenclature is crucial for obtaining the `future_version` in the changelog.
+Your attention to this specific branch naming convention is essential for accurate version tracking in the changelog.
+
+```shell
+export RELEASE_VERSION="X.Y.Z"
+git switch main
+git pull -r
```

Comment:
This should go in the modulesync repo, but I always prefer long options in scripts/docs because they're self-documenting.
```suggestion
git pull --rebase
```